README.md

```diff
@@ -115,6 +115,7 @@ We recommend looking at the [Example usage section](#example-usage) to understan
 | state_map | [state map object](#state-map-object) |  | v0.8.0 | List of entity states to convert (order matters as position becomes a value on the graph).
 | value_factor | number | 0 | v0.9.4 | Scale value by order of magnitude (e.g. convert Watts to kilo Watts), use negative value to scale down.
 | logarithmic | boolean | `false` | v0.10.0 | Use a Logarithmic scale for the graph
+| grid_line_type | string |  | v0.12.x | Show grids lines using `hour` / `day` / `week` and the value of hours_to_show
```

Comment:
Let's move it inside a `show` option?

Reply:
Also, is it possible to add `5minute` here? Note that `hours_to_show` could be fractional and < 1.